addons/web/static/src/core/commands/command_palette.variables.scss

```diff
@@ -0,0 +1,8 @@
+$o-command-palette-margin-top: 10vh;
+$o-command-palette-search-border: 1px solid rgba(0, 0, 0, 0.1);
+$o-command-palette-search-font-size: 1.25rem;
```

Comment:
This shouldn't be necessary. https://getbootstrap.com/docs/4.3/components/forms/#sizing